pkgs/development/python-modules/pytesseract/default.nix

```
@@ -37,6 +37,10 @@ buildPythonPackage rec {
     packaging
     pillow
   ];
+  disabledTests = [
+    "incorrect_tessdata_dir"
+    "invalid_tessdata_dir"
+  ];
```

```suggestion
   disabledTests = [
    # https://github.com/madmaze/pytesseract/pull/559
    "incorrect_tessdata_dir"
    "invalid_tessdata_dir"
  ];
```

Should link to upstream conversation around issues seen.